content/en/blog/2021/patch-tuesdays/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How security releases (patch tuesday, embargoes, 0-days) are handled by the Product Security working group
```

Comment:
Nit (can always push another PR later): Patch Tuesday (probably capitalize both). Not sure if Working Group would be capitalized as well.